examples/lessons/1/index.js

```diff
@@ -39,17 +37,18 @@ window.webGLStart = function() {
 
       gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT);
       camera.view.id();
+
       //Draw Triangle
       camera.view.$translate(-1.5, 0, -7);
-      program.setUniform('uMVMatrix', camera.view);
-      program.setUniform('uPMatrix', camera.projection);
+      program.uniforms.uMVMatrix(camera.view);
```

Comment:
This is as I found it in the es6 branch. I'm happy to refactor; was just trying to minimize my changes.